packages/cli/src/__tests__/update-message.test.ts

```diff
@@ -5,73 +5,157 @@ import { printUpdateMessage } from '../utils/printUpdateMessage'
 const ctx = jestContext.new().add(jestConsoleContext()).assemble()
 
 describe('update available message', () => {
-  it('dev tag - minor', () => {
+  let originalPrismaHideUpdateMessageEnv: string | undefined
+
+  beforeEach(() => {
+    originalPrismaHideUpdateMessageEnv = process.env.PRISMA_HIDE_UPDATE_MESSAGE
+    delete process.env.PRISMA_HIDE_UPDATE_MESSAGE
+  })
+
+  afterEach(() => {
+    process.env.PRISMA_HIDE_UPDATE_MESSAGE = originalPrismaHideUpdateMessageEnv
+  })
+
+  describe('update available', () => {
```

Comment:
ℹ️ Github diff is a bit stupid here: These tests already existed, I just nested them into a describe block. The tests below are the actual new ones.